Content.Client/Crayon/CrayonComponent.cs

```diff
@@ -10,5 +10,8 @@ public sealed partial class CrayonComponent : SharedCrayonComponent
         [ViewVariables(VVAccess.ReadWrite)] public bool UIUpdateNeeded;
         [ViewVariables] public int Charges { get; set; }
         [ViewVariables] public int Capacity { get; set; }
+        [ViewVariables] public string State { get; set; }
+        [ViewVariables] public float Rotation { get; set; }
+        [ViewVariables] public bool PreviewMode { get; set; }
```

Comment:
```suggestion
        [DataField] public string State;
        [DataField] public float Rotation;
        [DataField] public bool PreviewMode;
```
no need to write out get/set, also we use DataFields these days